### ISSUE TITLE
Managed Cloud SDK installation may fail

### ISSUE THREAD

**Issue by chanseokoh** [GoogleCloudPlatform/appengine-plugins-core#541]
It downloads an archive from https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz.

As of now, the version it downloads is 190.0.0. You can see the version reported while installing it (and also looking into its `VERSION` file after extracting the archive). However, 190.0.0 is actually newer than the current official latest, which is 189.0.0. Probably because of this, installation fails while trying to initiate "downgrading."

```
Downloading https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz
Downloading 9257599 bytes
.........done.
   (... omitted ...)
Your current Cloud SDK version is: 190.0.0
Installing components from version: 189.0.0

+-------------------------------------------------+
|        These components will be updated.        |
+--------------------------+------------+---------+
   (... omitted ...)
Performing in place update...

#============================================================#
#= Uninstalling: Cloud SDK Core Libraries                   =#
#============================================================#
   (... omitted ...)
Traceback (most recent call last):
  File "/usr/local/google/home/chanseok/.google-cloud-tools-java/managed-cloud-sdk/LATEST/google-cloud-sdk/bin/bootstrapping/install.py", line 218, in <module>
   (... omitted ...)
IOError: [Errno 2] No such file or directory
```

It is not clear if this is a temporary breakage, or it is possible that `https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz` may be newer than the official latest. @markpell?

**Comment by loosebazooka** [comment]
@vilasj

**Comment by chanseokoh** [comment]
Oh, just noticed that https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz has advanced its version from 190.0.0 to 190.0.1.

**Comment by chanseokoh** [comment]
190.0.1 works fine. I guess it was a temporary breakage with 190.0.0.

**Comment by vilasj** [comment]
Yes, 190.0.1 was released as a hotfix for an issue found in 190.0.0.
